src/pages/Home/Map.js

```diff
@@ -326,14 +323,6 @@ class Map extends Component {
                     ) : (
                         ''
                     )}
-                    <Geocoder
```

Comment:
Yep. I apologize that I popped that in after commenting on Trello. I figured it would make a cleaner starting point for a fresh PR: in Friendsearch.js if we put a connection to Map.js getPostcode, to update the lat/long positioning of map.